modules/ztp-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-prep_{context}"]
+= Preparing the {sno} clusters for the image-based upgrade
+
+When you deploy the {lcao} on a cluster, a blank `imageBasedUpgrade` CR is automatically created.
+You use this CR to specify the image repository of the seed image and to move through the different stages.
+
+In the `Prep` stage, the {lcao} ensures that the target clusters are ready for the upgrade by checking if they meet certain conditions. The {lcao} also precaches any additional listed images on the target clusters from the seed image.
```

Comment:
It also checks that the seed image is compatible with the LCA operator deployed.